spec/views/catalog/_reviewer_action.html.erb_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "catalog/_reviewer_action.html.erb" do
+  let(:document) { instance_double(SolrDocument, :id => "1")}
+  before do
+    allow(view).to receive(:can?).with(:edit, document).and_return(permission_result)
+    render :partial => "catalog/reviewer_action.html.erb", :locals => {:document => document }
+  end
+  
+  context "when they can review the document" do
+    let(:permission_result) { true }
+    context "the document is ready for review" do
+      let(:document) { instance_double(SolrDocument, :id => "1", :reviewed => false)}
+      it "should have the review link" do
+        expect(rendered).to have_link("Mark as Reviewed")
```

Comment:
I'd recommend putting the text "Mark as Reviewed" into a `let` variable.  If the text changes, this spec will fail while the one below (with `expect(rendered)not_to have_link...`) will continue to pass.  Somebody will fix this spec, and possibly not the one below.  Then the spec below is no longer actually verifying that the link is missing - it's verifying that text which no longer exists continues not to exist.